mdtraj/core/trajectory.py

```diff
@@ -395,7 +395,11 @@ def load(filename_or_filenames, discard_overlapping_frames=False, **kwargs):
     top = topkwargs.pop("top", None)
     if top is None:
         top = filename_or_filenames[0]
-    kwargs["top"] = _parse_topology(top, **topkwargs)
+
+    # These topology formats do not support the 'top' keyword
+    # This is to prevent the loader from reading the topology twice.
+    if extension not in ['.h5', '.hdf5', '.mol2']:
```

Comment:
Does MDTraj support loading (I feel filthy putting these words together in succession) MOL2 trajectories?